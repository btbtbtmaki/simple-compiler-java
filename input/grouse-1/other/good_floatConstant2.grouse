main {
	imm v := -2.0000;
    imm w := 12.3e-2;
    imm x := -12.34567e4;
    imm y := -123.4567;
    imm z := 0.0e1;
    print v, w, x, y, z, nl;
}