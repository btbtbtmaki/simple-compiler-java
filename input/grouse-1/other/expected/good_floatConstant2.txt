-2 0.123 -123457 -123.457 0 
